contracts/mockContracts/MockDGXDex.sol

```diff
@@ -0,0 +1,187 @@
+pragma solidity ^0.4.18;
+
+//ask:
+//dst address change?
+//should init another token outside in test?
+
+// blocknumber + 5 >= current block number ? revert
+// signatures - skip for now.
+
+import "../ERC20Interface.sol";
+import "../Withdrawable.sol";
+import "./ecverify.sol";
+
+
+/// @title Mock Digix DEX
+/// @author Tal Baneth
+/// @dev a dummy contract that simulates Digix contract for purchase/sell with verified signatures. 
+contract MockDGXDEX is Withdrawable {
+    uint  constant internal G_TO_MG = (10**3);
+    uint  constant internal DGX_DECIMALS = (10**9);
+    
```

Comment:
we don't use this notation.
only camel back notations...
Dabashot :)s

Reply:
I do see you use this notation for constants.